modules/nw-networkpolicy-about.adoc

```diff
@@ -6,33 +6,34 @@
 
 = About network policy
```

Comment:
@jboxman Just wondering if this should be "About NetworkPolicy"? That seems to be how you're referring to it everywhere else.

Reply:
Yes, thanks!